components/README.md

```diff
@@ -5,3 +5,21 @@
 The components directory contains your Vue.js Components.
 
 _Nuxt.js doesn't supercharge these components._
+
```

Comment:
please don't change this file

Reply:
I think we don't put our wedance firebase information.
As you know, our repo is public, so everyone who can use and clone our repo can see and manage that.